### DISCUSSION TITLE
[Fixed] Custom JS Code no longer works

### DISCUSSION THREAD

**Discussion by cargeo** [ingrammicro/puzzle-publisher#71] [General]
Hello,

in the field "Custom JS Code" I used to have the following code to hide certain elements in the menu:
`
 $("#menu hr").hide(); 
$("#menu #embed").hide();
$("#menu #grid").hide(); 
`
 
This no longer works :(

Maybe the menu could be configured with checkboxes one day. That would be nice.

**Comment by MaxBazarov** [comment]
Sorry. Will fix it asap.

**Comment by MaxBazarov** [comment]
> Hello,
> 
> in the field "Custom JS Code" I used to have the following code to hide certain elements in the menu: `$("#menu hr").hide(); $("#menu #embed").hide(); $("#menu #grid").hide(); `
> 
> This no longer works :(

Fixed in 16.4.4

> Maybe the menu could be configured with checkboxes one day. That would be nice.

Good idea. I want to refresh menu navigation soon. Will also add settings for all items.
BTW, do you like the latest menu redesign with top switchers?

**Comment by cargeo** [comment]
Thanks for the fix.

> I want to refresh menu navigation soon. 

Cool. Very often I need a very minimalistic viewer menu  since my clients don't need to see options for grid, embed etc.

> BTW, do you like the latest menu redesign with top switchers?

Yes, looks good.
